types/sinon/index.d.ts

```diff
@@ -293,14 +291,10 @@ declare namespace Sinon {
         FakeXMLHttpRequest: SinonFakeXMLHttpRequest;
     }
 
-    interface SinonFakeServer {
+    interface SinonFakeServer extends SinonFakeServerOptions {
```

Comment:
Does this mean that `SinonFakeServer` will have optional fields for autoRespond, autoRespondAfter,...? That doesn't sound good. Couldn't we use TypeScript's `Partial` mapped type? All fields on `SinonFakeServerOptions` should be required and the `options` parameter should be of type `Partial<SinonFakeServerOptions>`

Reply:
The original types were wrong. These properties are optional already. In both the config object and the server instance. If you read the source of sinon's fake server, you'll see only the ones you pass get set, thus the others are undefined (optional).

Reply:
Ah ok, than it's fine, thanks for the clarification :+1:

Reply:
@nicojs i think you need to approve the pr so the bot will mark it as mergeable. can you take a look?

Reply:
how? I just reacted with a thumbs up 👍